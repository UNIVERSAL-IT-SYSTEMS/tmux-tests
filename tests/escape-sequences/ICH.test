#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
printf "\033[@a"
printf "\n\033[0@b"
printf "\n\033[1@c"
printf "\n\033[2@d"
printf "\na\033[@a"
printf "\nb\033[0@b"
printf "\nc\033[1@c"
printf "\nd\033[2@d"
printf "\nabcdef\010\010\010\010\033[@"
printf "\nabcdef\010\010\010\010\033[0@"
printf "\nabcdef\010\010\010\010\033[1@"
printf "\nabcdef\010\010\010\010\033[2@"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
