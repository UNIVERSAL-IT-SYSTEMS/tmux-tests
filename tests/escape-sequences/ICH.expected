a
b
c
d
aa
bb
cc
dd
ab cdef
ab cdef
ab cdef
ab  cdef












